eZ/Publish/Core/Limitation/LanguageLimitationType.php

```diff
@@ -255,6 +270,20 @@ private function evaluateVersionTarget(
         return $accessVote;
     }
 
+    private function evaluateLocationTarget(
+        DestinationLocationTarget $location,
+        APILimitationValue $value
+    ): ?bool {
+        $versionInfo = $this->tryLoadingVersionInfo($location->getTargetContentInfo());
+        if ($versionInfo === null) {
+            return self::ACCESS_DENIED;
+        }
+
+        return !array_diff($versionInfo->languageCodes, $value->limitationValues)
```

Comment:
Implicit casting from any type (here array) to boolean tends to be error prone, so to develop good habits, we should avoid them, even if in any specific case they would work.
```suggestion
        return empty(array_diff($versionInfo->languageCodes, $value->limitationValues))
```